Localization/sv-SE/papers-and-results.po

```diff
@@ -0,0 +1,32 @@
+msgid ""
+msgstr ""
+"FAH-Translate-Schema-Version: 1.0\n"
+"URL: https://foldingathome.org/papers-results/\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Source-Language: en-US\n"
+"Source-Version: 1.0\n"
+"Source-Last-Modified: 20200907\n"
```

Comment:
Please make sure you are using latest Source-Last-Modified. 
On the en-US version is:
Source-Last-Modified: 20200923